plugins/module_utils/crypto.py

```diff
@@ -418,7 +418,13 @@ def write_file(module, content, default_mode=None, path=None):
     Uses file arguments from module.
     '''
     # Find out parameters for file
-    file_args = module.load_file_common_arguments(module.params, path=path)
+    try:
```

Comment:
Nice that this found a bug straight away :)

Reply:
It's not really a bug, it's using a feature which is only available in Ansible 2.10 :)

Reply:
(For reference, it was added in ansible/ansible#61659.)